api/Controllers/EventController.cs

```diff
@@ -118,14 +135,26 @@ public async Task<IActionResult> PutEvent(EventDto newEventDto)
 		{
 			var newEvent = await _eventService.CreateEvent(newEventDto);
 			var putEvent = await _eventService.PutEvent(newEvent);
+
+			if (newEvent.AutoJoin)
+			{
+				var attendanceDto = new AttendanceDto();
+				attendanceDto.EventId = putEvent.Id;
+				attendanceDto.UserId = newEvent.HostId;
+				attendanceDto.JoiningDate = DateTime.UtcNow.ToLocalTime();
+				var newAttendance = _mapper.Map<Attendance>(attendanceDto);
+				await _attendanceService.CreateAttendance(_mapper.Map<AttendanceDto, Attendance>(attendanceDto));
+			}
```

Comment:
This code is duplicate with the one on line 109. I suggest extracting this to a new method.